pkg/standalone/standalone.go

```diff
@@ -20,6 +20,7 @@ import (
 	"os/exec"
 	"os/user"
 	"path"
+	"path/filepath"
 	path_filepath "path/filepath"
```

Comment:
Looks like duplicate import, can we please remove this?

Reply:
thanks! fixed